### ISSUE TITLE
DIA_SpecLib_Quant workflow not completed

### ISSUE THREAD

**Issue by hcadre** [Nesvilab/FragPipe#919]
I tried to process 2 DIA PASEF files (data type defined as either DIA or DIA-Quant - I tried both), together with 60 short DDA PASEF runs (fractions) as auxiliary files for spectral library building, using fragpipe v19.0, with default values ON in all fragpipe modules. In both instances the analysis was abrogated with this error message (6 tasks remaining):

![image](https://user-images.githubusercontent.com/16349515/206442558-520ace15-7ade-49cf-a54d-0cff639bf369.png)

The library.tsv file that could not be loaded is present in the lib folder, contains data (about 900MB) and can be opened by other programs. Also, with Spectronaut the same DIA and DDA files could be analyzed without any glitches, although it took quote a while. So the DIA and DDA files themselves appear not to be corrupted.

Any suggestions on how to proceed? I am grateful for all comments!

**Comment by fcyu** [comment]
Can you share the whole log file with us?

Thanks,

Fengchao

**Comment by hcadre** [comment]
Nope, my email was addressed to notifications@github.com. I try to attach them here again. Hope this will work. 
[log_2022-12-08_04-32-54.txt](https://github.com/Nesvilab/FragPipe/files/10186672/log_2022-12-08_04-32-54.txt)
[log_2022-12-08_12-25-08.txt](https://github.com/Nesvilab/FragPipe/files/10186673/log_2022-12-08_12-25-08.txt)

**Comment by hcadre** [comment]
Hi Fengchao, 

Good to know! Thank you very much for this advice! Since this it neither mentioned on the fragpipe DIA tutorial page (https://fragpipe.nesvilab.org/docs/tutorial_DIA.html) nor in the description for the DIA_SpecLib_Quant workflow (within fragpipe), it would be helpful if you could you provide this information by inserting a few words there.

All the best, 

Hannes

**Comment by fcyu** [comment]
Hi Hannes,

Thanks for the suggestion. I have updated the tutorial.

Best,

Fengchao
